front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -109,23 +109,37 @@ export const CommandMenu = () => {
         onValueChange={setSearch}
       />
       <StyledList>
-        <StyledEmpty>No results found.</StyledEmpty>
-        {!matchingCreateCommand && (
-          <StyledGroup heading="Create">
-            {commandMenuCommands
-              .filter((cmd) => cmd.type === CommandType.Create)
-              .map((cmd) => (
-                <CommandMenuItem
-                  key={cmd.label}
-                  to={cmd.to}
-                  label={cmd.label}
-                  Icon={cmd.Icon}
-                  shortcuts={cmd.shortcuts || []}
-                  onClick={cmd.onCommandClick}
-                />
-              ))}
-          </StyledGroup>
-        )}
+        {!matchingCreateCommand &&
+          !matchingNavigateCommand &&
+          commandMenuCommands.filter((cmd) => cmd.type === CommandType.Create)
+            .length < 1 &&
+          commandMenuCommands.filter(
```

Comment:
this looks be the same purpose as matchingNavigateCommand, could you update matchingNavigateCommand to also check for the label and then remove the check on line 116 that will be duplicated with matchingNavigateCommand